packages/catalog-process/test/db.test.ts

```diff
@@ -1317,11 +1350,22 @@ describe("database test", async () => {
 
     describe("clone descriptor", () => {
       it("should write on event-store for the cloning of a descriptor", async () => {
+        const copy = vi.spyOn(fileManager, "copy");
```

Comment:
To avoid a similar issue, I am adding a similar check inside the test of the `clone descriptor`, because that operation has to copy both interface and docs.